### PR TITLE
Plugins

### PR DESCRIPTION
**Pull request by djiamnot**
This is a prototype of a blender addon for attaching SATIE audio plugins to 3D objects in Blender editor.

### CODE REVIEW

File: plugins/satie4blender/__init__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
```
Comment (nicobou) [inline plugins/satie4blender/__init__.py]: License header ? (same for other python files)

File: plugins/satie4blender/control.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    print("{}'s satie ID cannot be empty", o.name)
+            else:
+                if o.satieID in synths:
+                    print(">>>>>> removing {} from {}".format(o.satieID, o.name) )
+                    toRemove = [x for x in props.synths if x.id == o.satieID]
+                    for i in toRemove:
+                        i.deleteNode()
+                        props.synths.remove(i)
+
+    print("<<<<<< synths ", props.synths)
```
Comment (nicobou) [inline plugins/satie4blender/control.py]: Are you sure you want to print ? debug mode ?

Comment (djiamnot) [inline plugins/satie4blender/control.py]: Most prints removed

File: plugins/satie4blender/README.org [hunk 3]
```diff
@@ -0,0 +1,2 @@
+* SATIE OSC
+A Blender addon to link Blender objects with SATIE (Spatial Audio Toolkit for Immersive Environments) https://github.com/nicobou/satie
```
Comment (nicobou) [inline plugins/satie4blender/README.org]: ??

Comment (djiamnot) [inline plugins/satie4blender/README.org]: désolé...

**Pull request merged**